SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -92,8 +92,16 @@ TMap<FString, TSet<FString>> PotentialSchemaNameCollisions;
 // QBI
 TMap<float, Worker_ComponentId> NetCullDistanceToComponentId;
 
-const FString RelativeSchemaDatabaseFilePath = FPaths::SetExtension(
-	FPaths::Combine(FPaths::ProjectContentDir(), SpatialConstants::SCHEMA_DATABASE_FILE_PATH), FPackageName::GetAssetPackageExtension());
+namespace
+{
+const FString& GetRelativeSchemaDatabaseFilePath()
```

Comment:
Interesting, are there potentially other cases of this? I guess depending on init order we might discover them over time

Reply:
Could potentially switch to TCHAR* and avoid runtime construction altogether!

Reply:
Yeah, I'm going to look through the code to see if there are other instances.
This one jumped at me pretty early. But the schema generation code is notoriously badly setup.